ide

```diff
@@ -1 +1 @@
-Subproject commit 372252ac065560dec60c74e823a228eb2925b5ad
+Subproject commit 504860b26826acade8955ca49377993913358eec
```

Comment:
I don't think this submodule pointer update is correct:
https://github.com/livecode/livecode-ide/commits/develop-8.0
https://github.com/livecode/livecode-ide/commits/504860b26826acade8955ca49377993913358eec